artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerConsumerImpl.java

```diff
@@ -1547,4 +1555,9 @@ public String getConnectionLocalAddress() {
    public String getConnectionRemoteAddress() {
       return this.session.getRemotingConnection().getTransportConnection().getRemoteAddress();
    }
+
+   @Override
+   public Subject getSubject() {
```

Comment:
Dont expose tbis

Reply:
if something needs it like a plugun the server can be passed jn at injt stage or just pass server